### ISSUE TITLE
Macroadenoma

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7185]
**Preferred term label:**
Macroadenoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
A macroadenoma is a noncancerous tumor that typically develops in the pituitary gland
PMID: 27974908

**Parent term (use hpo.jax.org/app)**
Pituitary corticotropic cell adenoma (HP:0008291)
Pituitary adenoma (HP:0002893).

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:96253

**Comment by pnrobinson** [comment]
Adding as
Pituitary macroadenoma
Def: A pituitary gland adenoma that is larger than 10mm.
Comment: Pituitary macroadenoma presents with mass effects and potentially hormonal deficiency or hormonal excess.
PMID: 32119338
